### PR TITLE
tests and CHANGELOG entry for PR 1653

### PR DESCRIPTION
**Pull request by fallwith**
here are some unit tests and a CHANGELOG entry for the community submission delivered by PR #1653

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # New Relic Ruby Agent Release Notes #
 
+  ## v8.14.0
+
+  Version 8.14.0 of the agent restores desired Capistrano based changelog lookup functionalty when a deployment is performed.
+
+  * **Deployment recipe: restore desired Capistrano based changelog lookup behavior**
```
Comment (kaylareopelle) [inline CHANGELOG.md:7]:
```suggestion
  * **Deployment Recipe: Restore desired Capistrano-based changelog lookup behavior**
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
 # New Relic Ruby Agent Release Notes #
 
+  ## v8.14.0
+
+  Version 8.14.0 of the agent restores desired Capistrano based changelog lookup functionalty when a deployment is performed.
```
Comment (kaylareopelle) [inline CHANGELOG.md:5]:
```suggestion
  Version 8.14.0 of the agent restores desired Capistrano-based changelog lookup functionalty when a deployment is performed.
```

File: test/new_relic/recipes/helpers/send_deployment_test.rb [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require_relative '../../../test_helper'
+require_relative '../../../../lib/new_relic/recipes/helpers/send_deployment'
+
+module Newrelic
```
Comment (kaylareopelle) [inline test/new_relic/recipes/helpers/send_deployment_test.rb:8]: We use a capital "R" for "Relic" in most situations, however, not in all situations. It probably doesn't matter here.

**Review by kaylareopelle — APPROVED** [review]

**Comment by kaylareopelle** [comment]
Whoops, spoke too soon. Looks like there's an error related to the new tests in Ruby 2.2: 

```
  4) Error:
Newrelic::SendDeploymentTest#test_fetch_changelog_initial_fetch_succeeds_and_using_scm_then_use_fetched_value:
ArgumentError: wrong number of arguments (3 for 2)
    /home/runner/work/newrelic-ruby-agent/newrelic-ruby-agent/test/new_relic/recipes/helpers/send_deployment_test.rb:29:in `block in test_fetch_changelog_initial_fetch_succeeds_and_using_scm_then_use_fetched_value'
    /home/runner/work/newrelic-ruby-agent/newrelic-ruby-agent/test/new_relic/recipes/helpers/send_deployment_test.rb:28:in `test_fetch_changelog_initial_fetch_succeeds_and_using_scm_then_use_fetched_value'
```

File: test/new_relic/recipes/helpers/send_deployment_test.rb (hunk 3 above)
Comment (fallwith) [reply test/new_relic/recipes/helpers/send_deployment_test.rb]: addressed with [bcebb46](https://github.com/newrelic/newrelic-ruby-agent/pull/1654/commits/bcebb464c4274404fe3433032555cac283da0755)

**Comment by fallwith** [comment]
> Whoops, spoke too soon. Looks like there's an error related to the new tests in Ruby 2.2:
> 
> ```
>   4) Error:
> Newrelic::SendDeploymentTest#test_fetch_changelog_initial_fetch_succeeds_and_using_scm_then_use_fetched_value:
> ArgumentError: wrong number of arguments (3 for 2)
>     /home/runner/work/newrelic-ruby-agent/newrelic-ruby-agent/test/new_relic/recipes/helpers/send_deployment_test.rb:29:in `block in test_fetch_changelog_initial_fetch_succeeds_and_using_scm_then_use_fetched_value'
>     /home/runner/work/newrelic-ruby-agent/newrelic-ruby-agent/test/new_relic/recipes/helpers/send_deployment_test.rb:28:in `test_fetch_changelog_initial_fetch_succeeds_and_using_scm_then_use_fetched_value'
> ```

@kaylareopelle addressed with [bcebb46](https://github.com/newrelic/newrelic-ruby-agent/pull/1654/commits/bcebb464c4274404fe3433032555cac283da0755), thank you!

**Review by kaylareopelle — APPROVED** [review]
Thanks, James!

**Pull request merged**